examples/python/printing-client/README.md

```diff
@@ -0,0 +1,29 @@
+# StellarStation example client - PrintingClient
+
+A simple example client that prints received telemetry and sends commands at a fixed interval. Shows
+the basics of writing code to integrate with the StellarStation API. Only works with [Fakeserver](../../fakeserver).
+
+
+## Before you begin
+### Prerequisites
+When you start with gRPC in Python, you need pip and gRPC. Ensure you have correctly installed those modules by following instructions in official [gRPC site](https://grpc.io/docs/quickstart/python.html#install-grpc-tools).
```

Comment:
Don't need to do these anymore since `pip install stellarstation` installs it all (can verify by running commands in `docker run -it --rm python:2`